types/hydrogen.js.flow

```diff
@@ -10,3 +10,8 @@ export type Message = {
   header: Object,
   content: Object,
 }
+
+export type KernelCommand = {
```

Comment:
Since the `KernelCommand` type is likely not used in a lot of files, I think the clearest thing is to declare it in `signal-list-view.js.js`.